_analyzers/token-filters/synonym.md

```diff
@@ -0,0 +1,277 @@
+---
+layout: default
+title: Synonym
+parent: Token filters
+nav_order: 420
+---
+
+# Synonym token filter
+
+The `synonym` token filter allows you to map multiple terms to a single term or create equivalence groups between words, improving search flexibility.
+
+## Parameters
+
+The `synonym` token filter can be configured with the following parameters.
+
+Parameter | Required/Optional | Data type | Description
+:--- | :--- | :--- | :--- 
+`synonyms` | Either `synonyms` or `synonyms_path` must be specified | String | A list of synonym rules defined directly in the configuration.
```

Comment:
Nit: Here and on the following line, because they are technically complete sentences, the "Either" sentences should end with a period (I believe, however, that we may not do this on the other pages).